packages/lesswrong/lib/collections/tagRels/fragments.ts

```diff
@@ -9,12 +9,7 @@ registerFragment(`
     userId
     tagId
     tag {
-      _id
-      name
-      slug
-      description {
-        htmlHighlight
-      }
+      ...TagFragment
```

Comment:
This has a significant performance impact (it changes the hover-cards so that they have to load the whole tag description, not just the highlight).

Reply:
I don't think this is actually that big a performance impact, but can create a new fragment for it.

Reply:
This gets loaded on post pages, and while tag descriptions aren't currently very long, they certainly have the potential to get very large. So, better to make a new fragment.